src/json-api-client.coffee

```diff
@@ -101,9 +102,9 @@ class JSONAPIClient extends Model
     if attributeTypeName?
       type._links[attributeName].type = attributeTypeName
 
-  handleError: ->
+  handleError: (err) ->
     # Override this as necessary.
-    Promise.reject arguments...
+    Promise.reject err
```

Comment:
yep, previous way preferred, I'll revert.
I did unnecessarily while attempting to understand flow of responses/rejections.